src/index.css

```diff
@@ -1,3 +1,6 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap');
+
 * {
```

Comment:
- [ ] [OPTIONAL] Kindly make sure to use the right background color for your app and add some padding for the `Add book` form so the app looks exactly as the provided design

![Screenshot_20230701_215549](https://github.com/derrick1451/bookstore/assets/25227022/67de124c-78f0-4db1-bd1a-98c18b946500)